services/block_service.go

```diff
@@ -186,7 +179,7 @@ func (s *BlockAPIService) populateTransaction(
 	return populatedTransaction, nil
 }
 
-// GetEthBlock returns a populated block at the *RosettaTypes.PartialBlockIdentifier.
+// Block returns a populated block at the *RosettaTypes.PartialBlockIdentifier.
```

Comment:
the method name is `GetEthBlock` - so let's keep it this way. We should add a linter to this repo which can detect this error - I though there was already?

Reply:
there is a linter and is not failing. hmm. need to fix that